contrib/vagrant-ci/centos-7-x64/Vagrantfile

```diff
@@ -43,10 +43,12 @@ SHELL
     nd.vm.hostname = no_deps_vm_name.to_sym
 
     nd.vm.provision "shell", inline: <<-SHELL
-yum -q -y install lmdb-devel pcre-devel openssl-devel libyaml-devel libxml-devel
+set -e
+yum -q -y install lmdb-devel pcre-devel openssl-devel libyaml-devel libxml2-devel
```

Comment:
Seems like the commit message should have some explanation. Was this working before? Was there a change from CentOS 6 to 7? Has this always been wrong?

Reply:
@vpodzime had mentioned that before with centos-6 when he finished this it was passing all tests.

We don't run this on a regular basis so not sure when it failed.

Checking online repos of centos6 it seems like libxml2-devel is present and not libxml-devel so maybe a change in centos6 recently as well.

Reply:
I think maybe I was just lucky and something pulled in `libxml2-devel` too or it was already in the CentOS 6 box I used. No idea.